### DISCUSSION TITLE
whole_body.move_end_effector_pose() gives FollowTrajectoryError when target is more than 1.1 meter

### DISCUSSION THREAD

**Discussion by hansenmaster** [HumanSupportRobot/QandA#63] [Q&A]
We are using HSRB to grasp a tennis ball around 1.1 meter from base.
The target frame looks perfect on rviz, we use whole_body.move_end_effector_pose() to the target frame.

At starts robot moves toward the object while adjusting the arm, but when it's closer the both the base and arm stop midway and abort the mission.

The error terminal looks like this:
![image](https://github.com/HumanSupportRobot/QandA/assets/36762228/82619fb8-be33-48a9-9518-8bd9fa142ac1)

I understand that the omni_base_controller status is abort . 
There is no obstacle from robot to object, but this frequently happens when object is located further than 1 m
Does anyone also encounter this problem and know the turn around? 

 Thank you very much.

**Comment by k-takeshita** [comment]
It might be a deviation error in the base movement.
By writing the following parameters in /etc/opt/tmc/robot/params.yaml, there is a possibility to avoid stopping.

```
/hsrb/omni_base_controller/constraints/odom_x/trajectory: 0.0
/hsrb/omni_base_controller/constraints/odom_y/trajectory: 0.0
/hsrb/omni_base_controller/constraints/odom_t/trajectory: 0.0
```
